modules/oe_content_news/config/install/core.entity_form_display.node.oe_news.default.yml

```diff
@@ -30,51 +68,61 @@ content:
       rows: 9
       placeholder: ''
       summary_rows: 3
+      show_summary: false
     third_party_settings: {  }
     region: content
   created:
     type: datetime_timestamp
-    weight: 14
+    weight: 19
     region: content
     settings: {  }
     third_party_settings: {  }
+  langcode:
+    type: language_select
+    weight: 12
+    region: content
+    settings:
+      include_locked: true
+    third_party_settings: {  }
```

Comment:
I think also this shouldn't be exported in the default view mode, what happens when we enable this module on a non multilingual site? Let's check this in the follow up that we are  going to open above.